fetch.bs

```diff
@@ -4219,9 +4251,6 @@ objects.</span>
 "<code>response</code>" or
 "<code>none</code>".
 
-<p class="note no-backref">"<code>immutable</code>" exists for service workers.
-[[!SW]]
```

Comment:
Yeah, "immutable" is also used for `fetch()` responses. I could add the comment back and include the other places "immutable" is used.